src/components/Card/CardItem.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { Card } from 'antd';
```

Comment:
импорты можно переименовывать
```suggestion
import { Card as CardAntd } from 'antd';
```
теперь имя Card свободно и лучше наш компонент назвать просто Card